include/glow/Graph/Context.h

```diff
@@ -71,6 +71,10 @@ class Context final {
   /// tensors.
   void clear();
 
+  /// Sets \p cloned to be a copy of the Context, with each placeholder mapped
+  /// to a new Tensor, with their own memory.
+  void clone(Context &cloned) const;
```

Comment:
typically clone() just returns cloned object. What's the benefit of passing it by ref?

Reply:
Can we make Context movable?  I don't see a good reason why not, and it makes the type more ergonomic.